app/src/features/projects/view/components/FundingSource.tsx

```diff
@@ -136,89 +124,78 @@ const FundingSource: React.FC<IProjectFundingProps> = (props) => {
         onSave={handleDialogEditSave}
       />
       <ErrorDialog {...errorDialogProps} />
-      <Grid container spacing={3}>
-        <Grid container item xs={12} spacing={3} justify="space-between" alignItems="center">
-          <Grid item>
-            <Typography variant="h3">Funding Sources</Typography>
-          </Grid>
-          <Grid item>
-            <Button
-              variant="outlined"
-              component="label"
-              size="medium"
-              color="primary"
-              className={classes.addButton && classes.heading}>
-              Add Funding Source
-            </Button>
-          </Grid>
-        </Grid>
-        {funding.fundingSources.map((item: any, index: number) => (
-          <Fragment key={item.id}>
-            <Grid container item>
-              <Divider className={classes.topBorder} />
-            </Grid>
-            <Grid container item spacing={0} xs={12} justify="space-between" alignItems="center">
-              <Grid item xs={12} sm={6} md={4}>
-                <Box>
-                  <Typography className={classes.heading}>{item.agency_name}</Typography>
-                </Box>
-              </Grid>
-              <Grid item>
-                <Button
-                  className="editButtonSmall"
-                  onClick={() => handleDialogEditOpen(index)}
-                  title="Edit Funding Source Information"
-                  aria-label="Edit Funding Source Information"
-                  startIcon={<Icon path={mdiPencilOutline} size={0.875} />}>
-                  EDIT
-                </Button>
-              </Grid>
-            </Grid>
-            <Grid container item spacing={3} xs={12}>
-              <Grid item xs={12} sm={6} md={4}>
-                <Box color="text.disabled">
-                  <Typography variant="caption">Agency Project ID</Typography>
-                </Box>
-                <Box>
-                  <Typography style={{ wordBreak: 'break-all' }} variant="subtitle1">
+
+      <Box display="flex" alignItems="center" justifyContent="space-between" mb={2}>
+        <Typography variant="h3">Funding Sources</Typography>
+        <Button variant="outlined" color="primary">
+          Add Funding Source
+        </Button>
+      </Box>
+
+      {funding.fundingSources.map((item: any, index: number) => (
```

Comment:
think we need to use `hasFundingSources` here cause if `funding.fundingSources` is null or undefined this will fail

```
{hasFundingSources && funding.fundingSources.map((item: any, index: number) => (
```